src/components/VisualTesterKeymap.vue

```diff
@@ -181,7 +190,13 @@ export default {
         msg.push(`in ${(endTS - this.timing[ev.code]).toFixed(3)}ms`);
       }
       msg.unshift(
-        `Event key: '${ev.key}', Code: ${ev.code}, keyCode: ${ev.keyCode}`
+        `Event key: <span class="log-green">${ev.key.padEnd(
```

Comment:
let's make this a function since we're doing it so much:
```javascript
function greenMarkup(text, padlen) {
  return `<span class="log-green">${text.padEnd(padlen, ' ')}</span>`;
}
```

Reply:
function name is only a suggestion, pick something that makes more sense if you don't like it.